geopyspark/geotrellis/layer.py

```diff
@@ -454,9 +469,11 @@ def to_png_rdd(self, color_map):
         return create_python_rdd(result, ser)
 
     def to_geotiff_rdd(self,
-                       storage_method=StorageMethod.STRIPED,
+                       storage_method=StorageMethod.TILED,
                        rows_per_strip=None,
                        tile_dimensions=(256, 256),
+                       resample_method=ResampleMethod.NEAREST_NEIGHBOR,
+                       decimations=[2, 4, 8, 16],
```

Comment:
@jbouffard I feel like these should be off by default. This is extra work and storage requirement that may or may not be helpful and won't always be expected.